src/log/broadcast_backend.cr

```diff
@@ -0,0 +1,33 @@
+# A backend that broadcast to others backends.
+# Each of the referenced backends may have a different severity level filter.
+#
+# When this backend level is set that level setting takes precedence
+# over the severity filter of each referenced backend.
+#
+# This backend is not to be used explictly. It used by `Log::Builder` configuration
+# to allow a given source to emit to multiple backends.
+class Log::BroadcastBackend < Log::Backend
```

Comment:
The IO implementation with similar (broadcast) behaviour is called `MultiWriter`. Should we consider naming this one `MultiBackend` for consistency? `BroadcastBackend` seems more specific, though. So it might be a better choice after all.

Reply:
I think broadcast is a better description. :-) Although it is unlikely to be used publicly in the current state of the builder.